src/components/tick.js

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+
+export default class Tick extends React.Component {
+  static propTypes = {
+    angle: React.PropTypes.number,
+    x: React.PropTypes.number,
+    y: React.PropTypes.number,
+    length: React.PropTypes.number
+  };
+  static defaultProps = {
+    length: 6
+  };
+  render() {
+    const {angle, x, y, length} = this.props;
+
+    return (
+      <g>
+        <line
+          transform={`rotate(${angle}, ${x}, ${y})`}
+          stroke="black"
```

Comment:
these style attrs should also going in a style object. We need to be able to apply arbitrary passed styles provided by the user.